bin/autodeploy.sh

```diff
@@ -134,12 +136,22 @@ system_cleanup() {
   rm -f /home/$DEMO_USER/wire-server-deploy-static-*.tgz
 }
 
+# Make sure the host machine is up to date and has all required packages installed
+ssh -p "$SSH_PORT" "$SSH_USER"@webapp."$TARGET_SYSTEM" "
+  sudo apt update && sudo apt upgrade -y && sudo apt install -y aptitude apt-transport-https bind9-host curl debian-goodies dnsutils git dnsmasq less lsof net-tools rsyslog screen sudo vim wget whois docker.io telnet python3-lxml qemu qemu-kvm qemu-utils libvirt-clients libvirt-daemon-system virtinst bridge-utils
```

Comment:
I think we should let the Hetzner ansible playbook take care this.

You want to make sure that the `system_cleanup()` function does not fail when trying to run the script on a clean host, right?

I would create an `if` branch within that function to determine if the commands are available. Something like this:

```
system_cleanup() {
  if which docker > /dev/null && which virsh > /dev/null
  then
    for VM in $(virsh list --all --name); do virsh destroy "$VM"; virsh undefine "$VM" --remove-all-storage; done
    docker system prune -a -f
  fi
  rm -f /home/$DEMO_USER/.ssh/known_hosts
  rm -rf /home/$DEMO_USER/wire-server-deploy
  rm -f /home/$DEMO_USER/wire-server-deploy-static-*.tgz
}
```

Reply:
@jschumacher-wire I have updated the script to handle more cases, PTAL